<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageDock</class>
 <widget class="QWidget" name="ImageDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>870</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0">
    <widget class="QLabel" name="lHeight">
     <property name="text">
      <string>Height:</string>
     </property>
    </widget>
   </item>
   <item row="20" column="5">
    <widget class="QComboBox" name="cbBorderStyle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="ResizableTextEdit" name="teComment"/>
   </item>
   <item row="19" column="0" colspan="3">
    <widget class="QLabel" name="lBorder">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Border</string>
     </property>
    </widget>
   </item>
   <item row="25" column="0" colspan="4">
    <widget class="QCheckBox" name="chbVisible">
     <property name="text">
      <string>Visible</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="4">
    <widget class="QLabel" name="lVerticalAlignment">
     <property name="text">
      <string>Vert. align.:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="23" column="5">
    <widget class="QSpinBox" name="sbBorderOpacity">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The opacity ranges from 0 to 100, where 0 is fully transparent and 100 is fully opaque.</string>
     </property>
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="17" column="5">
    <widget class="QSpinBox" name="sbRotation">
     <property name="suffix">
      <string>Â°</string>
     </property>
     <property name="minimum">
      <number>-360</number>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="15" column="5">
    <widget class="QComboBox" name="cbHorizontalAlignment"/>
   </item>
   <item row="24" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>128</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QLabel" name="lSize">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Size</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>58</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="21" column="5">
    <widget class="KColorButton" name="kcbBorderColor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QSpinBox" name="sbOpacity">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>The opacity ranges from 0 to 100, where 0 is fully transparent and 100 is fully opaque.</string>
     </property>
     <property name="suffix">
      <string> %</string>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="21" column="0" colspan="4">
    <widget class="QLabel" name="lBorderColor">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item row="22" column="5">
    <widget class="QDoubleSpinBox" name="sbBorderWidth">
     <property name="suffix">
      <string> pt</string>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="16" column="5">
    <widget class="QComboBox" name="cbVerticalAlignment"/>
   </item>
   <item row="0" column="5">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="13" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QComboBox" name="cbPositionX">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Horizontal position relative to label's parent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbPositionX">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Horizontal position relative to label's parent</string>
       </property>
       <property name="suffix">
        <string> cm</string>
       </property>
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="lComment">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Comment:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>58</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="18" column="0">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>58</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QLabel" name="lOpacity">
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lRatio">
     <property name="text">
      <string>Keep Ratio:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lPosition">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="9" column="5">
    <widget class="QDoubleSpinBox" name="sbHeight">
     <property name="suffix">
      <string>cm</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="lName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lPositionY">
     <property name="toolTip">
      <string>Vertical position relative to label's parent</string>
     </property>
     <property name="text">
      <string>Position Y:</string>
     </property>
    </widget>
   </item>
   <item row="17" column="0" colspan="4">
    <widget class="QLabel" name="lRotation">
     <property name="text">
      <string>Rotation:</string>
     </property>
    </widget>
   </item>
   <item row="22" column="0" colspan="3">
    <widget class="QLabel" name="lBorderWidth">
     <property name="text">
      <string>Width:</string>
     </property>
    </widget>
   </item>
   <item row="23" column="0" colspan="3">
    <widget class="QLabel" name="lBorderOpacity">
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="5">
    <widget class="QCheckBox" name="chbKeepRatio">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="14" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QComboBox" name="cbPositionY">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Vertical position relative to label's parent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sbPositionY">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Vertical position relative to label's parent</string>
       </property>
       <property name="suffix">
        <string> cm</string>
       </property>
       <property name="minimum">
        <double>-99.989999999999995</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lWidth">
     <property name="text">
      <string>Width:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="lPositionX">
     <property name="toolTip">
      <string>Horizontal position relative to label's parent</string>
     </property>
     <property name="text">
      <string>Position X:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <widget class="QDoubleSpinBox" name="sbWidth">
     <property name="suffix">
      <string>cm</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="4">
    <widget class="QLabel" name="lHorizontalAlignment">
     <property name="text">
      <string>Hor. align.:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QLabel" name="lFileName">
     <property name="text">
      <string>File name:</string>
     </property>
    </widget>
   </item>
   <item row="20" column="0" colspan="2">
    <widget class="QLabel" name="lBorderStyle">
     <property name="text">
      <string>Style:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lEmbedded">
     <property name="text">
      <string>Embedded:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QCheckBox" name="chbEmbedded">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="leFileName">
       <property name="toolTip">
        <string>Specify the name of the image file</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bOpen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string> Select the image file</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>src/kdefrontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
